packages/grafana-ui/src/components/Logs/LogDetails.test.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import { Field, GrafanaTheme2, LogLevel, LogRowModel, MutableDataFrame } from '@grafana/data';
+
 import { LogDetails, Props } from './LogDetails';
-import { LogRowModel, LogLevel, MutableDataFrame, Field, GrafanaTheme2 } from '@grafana/data';
-import { mount } from 'enzyme';
-import { LogDetailsRow } from './LogDetailsRow';
+import { expect } from '../../../../../public/test/lib/common';
```

Comment:
@hugohaggmark just noticed this - probably IDE adding an import

Reply:
arghhh